x-pack/plugins/observability_solution/investigate_app/server/models/investigation.ts

```diff
@@ -5,22 +5,8 @@
  * 2.0.
  */
 
-import { alertOriginSchema, blankOriginSchema } from '@kbn/investigation-shared';
+import { investigationSchema } from '@kbn/investigation-shared/src/schema';
```

Comment:
Can we import like this `import { investigationSchema } from '@kbn/investigation-shared';` and export `schema` folder directly in the index file of the kbn-investigation-shared package?